2017-2018/GameEngine/pacbot/gameState.py

```diff
@@ -158,4 +168,6 @@ def restart(self):
         self.state_counter = 0
         self.update_ticks = 0
         self.lives = starting_lives
+        self.elapsed_time = 0
+        self.previous_start = time.time()
```

Comment:
Isn't this unnecessary since the game starts in a paused state?

Reply:
indeed